docs/quickstart/create-and-publish-a-package-using-visual-studio.md

```diff
@@ -18,9 +18,11 @@ It's a simple process to create a NuGet package from a .NET Standard Class Libra
 
 1. Install any edition of Visual Studio 2017 from [visualstudio.com](https://www.visualstudio.com/) with any .NET-related workload. Visual Studio 2017 automatically includes NuGet capabilities when a .NET workload is installed.
 
-1. Install the `nuget.exe` CLI by downloading it from [nuget.org](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe), saving that `.exe` file to a suitable folder, and adding that folder to your PATH environment variable.
+1. Install one of the NuGet CLI tools.
```

Comment:
The NuGet CLI refers to nuget.exe specifically, so I suggest changing this to "Install of the CLI tools" to keep it generic to mean either nuget.exe or dotnet.exe

Reply:
Correcting